Point_set_processing_3/examples/Point_set_processing_3/write_ply_points_example.cpp

```diff
@@ -32,7 +32,7 @@ namespace CGAL {
       if (is_ascii(out))
         out << int(c[0]) << " " << int(c[1]) << " " << int(c[2]) << " " << int(c[3]);
       else
-        CGAL::write(out, c);
+        out.write(reinterpret_cast<const char*>(&c), sizeof(c));
```

Comment:
That is really strange. Unless there is an overload of `CGAL::write` that we do not know, the two codes are equivalent.